src/GetSystemMetrics-Hook.cpp

```diff
@@ -19,23 +69,457 @@ int WINAPI Hooked_GetSystemMetrics(int nIndex)
         return Original_GetSystemMetrics(nIndex); // Don't override the other SystemMetrics requests
     }
 }
+// Implement MySetClipboardData which does nothing
+HANDLE WINAPI MySetClipboardData(UINT uFormat, HANDLE hMem)
+{
+    // This custom function does nothing
+    std::cout << "SetClipboardData hook called, but not setting clipboard data." << std::endl;
```

Comment:
I'm mostly against logging inside the hooked functions since it's really not very useful to know when a hooked function was called outside of debugging, and doing pretty much anything at all inside of a hook is somewhat risky. I wouldn't be opposed to keeping this inside of an 
```c
#ifdef DEBUG
std::cout << "stuff"
#endif
```
guard though.